session_2/student_registry/app/models/student.rb

```diff
@@ -1,2 +1,12 @@
+class CGPAValidator < ActiveModel::Validator
```

Comment:
You don't need to write your own validation here. A simple 
```ruby
validates :cgpa, numericality: {
  greater_than_equal_to: 0.0,
  less_than_equal_to_to: 10.0
}
```

Should work

https://guides.rubyonrails.org/active_record_validations.html#numericality

Reply:
hmm ok ty